apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -130,27 +130,23 @@ const TeacherApplication = React.createClass({
       whyCsIsImportant: isUnder1000Chars,
       whatTeachingSteps: isUnder1000Chars,
     };
-  },
-
-  getInitialState() {
-    return this.props.accountEmail ? {primaryEmail: this.props.accountEmail} : {};
```

Comment:
It looks like this state initializer wasn't recreated elsewhere in the component.  Intentional?

Reply:
Oh yeah, that also wasn't used. It must have been leftover from an earlier version, or development. The `primaryEmail` form control renders directly from `this.props.accountEmail`. See https://github.com/code-dot-org/code-dot-org/blob/dc5c967cc750e092a486311366ea472f9a6db21a/apps/src/code-studio/pd/teacher_application/teacher_application.jsx#L259-L267